source/SH3/system/glprogram.cpp

```diff
@@ -48,6 +48,44 @@ std::string sh3_glprogram::load_error::message() const
     return error;
 }
 
+
+void sh3_glprogram::Load(const std::string& name)
```

Comment:
I think OpenGL will crash if I specify no attributes, I'll have to double check. I need to research a bit more to better abstract some of this stuff.